integration-cli/docker_cli_daemon_test.go

```diff
@@ -106,12 +106,9 @@ func (s *DockerDaemonSuite) TestDaemonRestartWithVolumesRefs(c *testing.T) {
 		c.Fatal(err, out)
 	}
 
-	out, err := s.d.Cmd("inspect", "-f", "{{json .Mounts}}", "volrestarttest1")
+	out, err := s.d.Cmd("inspect", "-f", `{{range .Mounts}}{{if eq .Destination "/foo"}}FOUND{{end}}{{end}}`, "volrestarttest1")
 	assert.NilError(c, err, out)
```

Comment:
It'd be handy if the test listed all the mounts that were present if the assertion fails.
```suggestion
	out, err := s.d.Cmd("inspect", "-f", `{{range .Mounts}}{{.Destination}}{{'\0'}}{{end}}`, "volrestarttest1")
	assert.NilError(c, err, out)
	assert.Contains(c, strings.Split(out, '\0'), "/foo")
```

Reply:
Good idea. will add